entities/markets/trade.ts

```diff
@@ -36,6 +36,15 @@ export const tradesCollateralAmountSpent = ({ fpmmTrades }: TradesProps) => {
   );
 };
 
+export const tradesOutcomeTokensTraded = ({ fpmmTrades }: TradesProps) => {
+  return (
+    fpmmTrades?.reduce((acc, trade) => {
+      const type = trade.type;
+      return valueByTradeBigInt[type](acc, BigInt(trade.outcomeTokensTraded));
+    }, BigInt(0)) ?? BigInt(0)
+  );
+};
+
```

Comment:
I don't think `tradesOutcomeTokensTraded` is a good name.

It return the total, right? Not the array of trades. So I think we should add `total` or `sum` to the name.

Maybe something like `outcomeTokensTradedTotal` would describe better